packages/api-fetch/README.md

```diff
@@ -17,9 +17,22 @@ _This package assumes that your code will run in an **ES2015+** environment. If
 ```js
 import apiFetch from '@wordpress/api-fetch';
 
+// GET
 apiFetch( { path: '/wp/v2/posts' } ).then( posts => {
 	console.log( posts );
 } );
+
+// POST
+apiFetch( {
+	path: '/wp/v2/posts/1',
+	method: 'POST',
+	headers: {
+		'Content-Type': 'application/json',
+	},
+	body: JSON.stringify( { title: 'New Post Title' } ),
```

Comment:
I'm pretty sure you can drop the headers and the body here and replace with just:

```suggestion
	data:  { title: 'New Post Title' },
```

Reply:
Yes, good point. I was able verified that. Let's go with the terser solution.